apiserver/common/networkingcommon/types.go

```diff
@@ -36,6 +37,8 @@ type BackingSubnet interface {
 	Status() string
 	SpaceName() string
 	SpaceID() string
+	Refresh() error
```

Comment:
I'd rather not add this here, opting instead for a tightly focussed interface at the usage site.

Reply:
The latest patch d983087c9190e5e77b43920e4bb5f9d4f2c0c27b should fix that by using the needed Subnet interface defined at `move.go` itself.